### ISSUE TITLE
Icon request: Meetup "M" (Note: /not/ dup of #1588

### ISSUE THREAD

**Issue by secret-agent-girl** [FortAwesome/Font-Awesome#15394]
**Icon request checklist**
single icon
[done ] The title starts with "Icon request: " and is followed by the requested icon name (Ex: `Icon request: magical-unicorn`)
[ done] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate
[
https://www.google.com/imgres?imgurl=https%3A%2F%2Fcdn0.iconfinder.com%2Fdata%2Ficons%2Fsocial-network-8%2F50%2F52-512.png&imgrefurl=https%3A%2F%2Fwww.iconfinder.com%2Ficons%2F939820%2Fmeetup_icon&docid=dyY98XGY7EjWBM&tbnid=80ig0-hoMJmKaM%3A&vet=10ahUKEwiE2cqi7Y3kAhUFLa0KHWuOBCUQMwh2KAkwCQ..i&w=512&h=512&client=firefox-b-1-d&bih=729&biw=1462&q=meetup%20icon&ved=0ahUKEwiE2cqi7Y3kAhUFLa0KHWuOBCUQMwh2KAkwCQ&iact=mrc&uact=8
*or*
https://www.google.com/imgres?imgurl=https%3A%2F%2Fwww.shareicon.net%2Fdownload%2F2015%2F10%2F04%2F111962_meetup.svg&imgrefurl=https%3A%2F%2Fwww.shareicon.net%2Fmeetup-111962&docid=o-fTJeVV6t9-SM&tbnid=xk-3VgL3QCQyyM%3A&vet=10ahUKEwiE2cqi7Y3kAhUFLa0KHWuOBCUQMwh6KA0wDQ..i&w=800&h=800&client=firefox-b-1-d&bih=729&biw=1462&q=meetup%20icon&ved=0ahUKEwiE2cqi7Y3kAhUFLa0KHWuOBCUQMwh6KA0wDQ&iact=mrc&uact=8
 ] I have included at least one example image if it could be helpful (optional)
[yes ] The request is for a concrete object (it's harder to make an icon to represent happiness, it's easier to make a smiley face. ☺)

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

I'm taking a look at https://help.meetup.com/hc/en-us/articles/360001655932-Meetup-Trademark-Guidelines and it does not contain the icon in your example.

The request for an alternative version of the logo is legit, but we are also doing our best to respect official brands.

Is the one you are requesting an old logo of Meetup?

**Comment by secret-agent-girl** [comment]
Yes, it is. I'm not sure if I can find a link to demonstrate that, but I will look.

It is also a much clearer image for including social sharing 'bars' on websites.

**Comment by tagliala** [comment]
Ok, I've found this reference: https://www.underconsideration.com/brandnew/archives/new_logo_and_identity_for_meetup_by_sagmeister_walsh.php

**Comment by secret-agent-girl** [comment]
Thank you. I also found some snapshots on the Wayback Machine showing them using the style of the M I posted.

**Comment by sensibleworld** [comment]
So... our backlog of brand requests has reached a point where even we can't pretend we're keeping up. Some of these issues were opened before I had gray hair. And I've had gray hair for a LONG time. On top of that, treating requests as issues has always made it way too hard to see the forest for the trees.

To fix this, we are now managing all brand requests in **GitHub Discussions**. No more +1s or thumbs-ups needed. There's an upvote system built-in.

We are closing existing brand requests as part of this transition, but **we are not saying no** to your request.

**If this brand request is still relevant for you, please re-open it in Discussions** (and feel free to include a link back to this issue):
-> https://github.com/FortAwesome/Font-Awesome/discussions/new?category=brand-requests

And listen. We know this is annoying. If you need to vent, here are some pre-written responses you can use when opening the new discussion:

* Cool cool cool. No worry at all. I love re-requesting a brand that's redesigned three times while this issue was open.
* If an issue closes after seven years and nobody responds, did it ever really exist?
* I see you have chosen violence today. Re-opening per instructions.
* What's next? I have to file a TPS report, too?
* I am filled with the fury of a thousand suns. Why do I have to re-open the discussion? Couldn't you have just bulk converted all of the issues to discussions? What? You couldn't? What do you mean Github removed that feature earlier this year? You'd have to manually click the "convert to discussion" button on thousands of issues, while also ensuring it gets added to the correct category? That's a bunch of bull-phooey.
* No biggie.
